tests/fixtures/modules/invalid-with-statement-in-module.result.js

```diff
@@ -0,0 +1,6 @@
+module.exports = {
```

Comment:
well, really what this is testing is that `scriptType=module` should imply `script mode` where `with` declaration is forbidden.